src/Foundation/Speedy/serialization/SpeedySettings/Speedy/Speedy Global Settings.yml

```diff
@@ -7,26 +7,22 @@ DB: master
 SharedFields:
 - ID: "5a158a54-b524-483c-a3a0-6e2a5fcc5cbe"
   Hint: EndpointURL
-  Value: "https://nodeapicritical.azurewebsites.net/criticalfonts/"
+  Value: "http://localhost:3000"
 - ID: "af9fa73f-f0a8-4d58-8dea-cb2169147c87"
   Hint: DeferJSLoadForMilliseconds
   Value: 6000
 - ID: "f4006c48-6487-43b7-8d65-a3c32c637e29"
   Hint: RemoteFontMap
   Value: |
-    { "fontmap": [
+    { "fontMap": [
         {
-          "find": "../../local/Temp/fonts/national2web/",
-          "replace": "/-/media/Themes/RAA/Web/Web/fonts/national2web/"
+          "find": "/Users/sitecore/AppData/Local/fonts/tahoma/",
+          "replace": "/-/media/themes/anziif/anziif-theme/fonts/tahoma/"
         },
```

Comment:
The font map is just an example it changes for each website dependent on what they use.

Reply:
Ah unless this replacement matches the new habitat example.  That would make sense :)

Reply:
Ah ok its for anziif ... we may want to roll that back for github.